libsession/src/main/java/org/session/libsession/messaging/messages/control/ClosedGroupControlMessage.kt

```diff
@@ -170,7 +170,7 @@ class ClosedGroupControlMessage() : ControlMessage() {
                 }
                 is Kind.EncryptionKeyPair -> {
                     closedGroupControlMessage.type = DataMessage.ClosedGroupControlMessage.Type.ENCRYPTION_KEY_PAIR
-                    closedGroupControlMessage.publicKey = kind.publicKey
+                    closedGroupControlMessage.publicKey = kind.publicKey ?: ByteString.EMPTY
```

Comment:
The issue is when kind.publicKey is null, it throws a NullPointerException, so I left that here as a safety in case there is null, but this case shouldn't happen.